report_webkit_barcode/__init__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
Please use short headers like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py